azext_iot/digitaltwins/params.py

```diff
@@ -456,6 +456,14 @@ def load_digitaltwins_arguments(self, _):
             "In case of an error processing a batch, the behavior is determined by the --failure-policy parameter. ",
             arg_group="Models Input",
         )
+        context.argument(
+            "max_models_per_batch",
+            options_list=["--max-models-per-batch", "--mmpb"],
+            type=int,
+            help="The maximum model size per batch when creating more than 250 models."
+            "Reduce this number to prevent a DTDLParser error. ",
```

Comment:
q - if the number of models is less than 250, will this input be ignored?

Ex: there are 120 models and the mmpb is set to 30 -> will there be one batch of 100 or 4 batches of 30?

Reply:
yeah this 30 will be ignored and not use if the number of models is less than 250😀

Reply:
Either add a help example to show this or add in "This number will be ignored if the number of models is less than 250". Sort of leaning towards help examples b/c I hate long param descriptions

Also is there a test for this

Reply:
should I still add example in help if we hide the parameter to user?😀

Reply:
and good idea I will add a unit test for the non default value, thanks!